packages/instrumenter/.vscode/launch.json

```diff
@@ -16,10 +16,7 @@
       "args": [
         "--no-timeout"
       ],
-      "env": {
-        // Set CHAI_JEST_SNAPSHOT_UPDATE_ALL variable when you want to update the snapshot files
-        // "CHAI_JEST_SNAPSHOT_UPDATE_ALL": "true"
-      }
```

Comment:
Should we change this if we want it merged into the stryker repo?

Reply:
Good observation, will do